server/migrations/20190603150654-create-article.js

```diff
@@ -0,0 +1,48 @@
+const Articles = {
```

Comment:
Go and change the name of this file to something more readable. Check Lewis [PR](https://github.com/andela/king-kong-ah-backend/pull/9)